syz-manager/stats.go

```diff
@@ -29,12 +31,40 @@ type Stats struct {
 	corpusCoverFiltered Stat
 	corpusSignal        Stat
 	maxSignal           Stat
+	syzExecTotal        prometheus.GaugeFunc // Prometheus float64.
```

Comment:
Don't prefix fields with "syz". All 100000 types, functions and fields in this code base are syz-something, it communicates nothing, but make identifiers longer and adds visual noise.

Reply:
Renaming these to `prom` prefixed. I want to keep these inside the stat struct because I would want to eventually migrate the older stat constructs to be replaced by these ones.